hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/InvocationRegistry.java

```diff
@@ -89,18 +91,17 @@ long getLastCallId() {
      * Registers an invocation.
      *
      * @param invocation The invocation to register.
+     * @return true if InvocationRegistry is alive when invocation is registered, false otherwise
```

Comment:
this slightly misleading as technically the invocation is always registered. 
perhaps there is not difference for practical purposes, feel free to ignore this comment.